lib/train/platforms/detect/specifications/os.rb

```diff
@@ -76,6 +76,13 @@ def self.load
             true
           end
         end
+      plat.name("kali").title("Kali Linux").in_family("debian")
+        .detect do
+          if linux_os_release && linux_os_release["ID"].eql?("kali")
+            @platform[:release] = linux_os_release["VERSION"]
```

Comment:
This is 3 calls to `linux_os_release`... please use local variables to make it only 1.

Reply:
Updated in latest PR